src/decoder.rs

```diff
@@ -86,5 +86,6 @@ pub fn build_imac_decoder<R: Register>(version: u32) -> Decoder {
     decoder.add_instruction_factory(rvc::factory::<R>);
     decoder.add_instruction_factory(i::factory::<R>);
     decoder.add_instruction_factory(m::factory::<R>);
+    decoder.add_instruction_factory(b::factory::<R>);
```

Comment:
Note the name here is `build_imac_decoder`, we should not introduce more extensions here. B should only be activated when someone explicitly specify it now.

Reply:
Understand. I am going to add an additional flag when the machine is initializing, might look like the following Machine(version=VERSION0, extension=FLAG_IMAC | FLAG_B)

Reply:
I submitted an update, but CI is broken until this PR merged: https://github.com/nervosnetwork/ckb-vm-test-suite/pull/2